examples/tensorflow/translation/run_translation.py

```diff
@@ -339,7 +339,7 @@ def main():
             use_auth_token=True if model_args.use_auth_token else None,
         )
     # See more about loading any type of standard or custom dataset (from files, python dict, pandas DataFrame, etc) at
-    # https://huggingface.co/docs/datasets/loading_datasets.html.
+    # https://huggingface.co/docs/datasets/v2.13.1/loading.
```

Comment:
```suggestion
    # https://huggingface.co/docs/datasets/main/loading
```

Reply:
Thanks for catching this! The only change I'd suggest is to use `main` rather than pinning to a specific version.